table/destination.go

```diff
@@ -522,6 +524,12 @@ func (dest *Destination) computeKnownBestPath() (*Path, BestPathReason, error) {
 		return nil, BPR_UNKNOWN, nil
 	}
 
+	if SelectionOptions.DisableBestPathSelection {
```

Comment:
Better to place this above L512?

Reply:
Thank your for reviewing. L512 is typo of L520? If so, yes, better to place there because no longer needed to access to `dest.knownPathList[0]`. I will update my patch.

Reply:
Oops, right, L520.

Reply:
Thank you for confirmation. I will fix it.